### PR TITLE
Refactor with date

### PR DESCRIPTION
**Pull request by Olamarx**
### Add Date and Modules
- Refactored the Awesome Books
- Add date to be functioning properly and locally

### CODE REVIEW

File: src/modules/localStorage.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+export default class Storage {
+  static takingFromStorage() {
```
Comment (TedLivist) [inline src/modules/localStorage.js:2]: - [ ] Your app does not load the data from the localStorage on page reload. Kindly fix that.

  olamarx

File: src/modules/library.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    bookButton.classList.add('remove-btn');
+  }
+
+  static removeBook(element) {
+    if (element.classList.contains('remove-btn')) {
+      element.parentNode.remove();
+    }
+  }
+
+  static clearInputs() {
```
Comment (TedLivist) [inline src/modules/library.js:31]: - [ ] In keeping with ES6 syntax kindly refactor your functions to be arrow functions.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Olamarx,

Good job so far!
✔️ Use of Luxon package
✔️ Books can be added
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Mre55 — APPROVED** [review]
**Hi @Olamarx,**

# Status: APPROVED 🏆 🎉 

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀: 
Congratulations! 🎉 

## To highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The commit messages are meaningful ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 

Alt Text


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**